tests/sentry/backup/__init__.py

```diff
@@ -12,114 +13,129 @@
     sorted_dependencies,
 )
 from sentry.backup.helpers import DatetimeSafeDjangoJSONEncoder, get_exportable_sentry_models
+from sentry.utils.json import JSONData
 
 
-def targets(expected_models: list[Type[models.Model]]):
-    """A helper decorator that checks that every model that a test "targeted" was actually seen in
-    the output, ensuring that we're actually testing the thing we think we are. Additionally, this
-    decorator is easily legible to static analysis, which allows for static checks to ensure that
-    all `__relocation_scope__ != RelocationScope.Excluded` models are being tested.
+def verify_models_in_output(expected_models: list[Type[models.Model]], actual_json: JSONData):
```

Comment:
this should be `-> None` (otherwise mypy doesn't opt this into type checking)

Reply:
Done.